handlers.go

```diff
@@ -210,6 +210,10 @@ func forwardEvent(falcopayload types.FalcoPayload) {
 		go webhookClient.WebhookPost(falcopayload)
 	}
 
+	if config.CloudEvents.Address != "" && (priorityMap[strings.ToLower(falcopayload.Priority)] >= priorityMap[strings.ToLower(config.CloudEvents.MinimumPriority)] || falcopayload.Rule == TestRule) {
+		go cloudeventsClient.CloudEventsPost(falcopayload)
```

Comment:
As a suggestion, why don't we make the name of the method that fires the event strict?

We use `Publish` for GCP PubSub, `Produce` for Kafka Producer, `Post` for Webhooks, API (e.g. `SlackPost`, `WebhookPost`), etc.
In the CloudEvent SDK, `Send` is used to fire events to the backend (ref: https://github.com/cloudevents/sdk-go/blob/93f9fa1cd3103d71d1b609a75ec3160ed601b436/v2/client/client.go#L22). 

Suggestion: 

```suggestion
		go cloudeventsClient.CloudEventsSend(falcopayload)
```

If this looks good, please rename the method in `outputs/cloudevent.go` too  🙇